app/src/features/authentication/login/hooks/useAuthCheck.ts

```diff
@@ -29,7 +29,7 @@ const useAuthCheck = (path: Path) => {
     }
 
     checkAuth();
-  }, [navigate]);
+  }, [navigate, path]);
```

Comment:
this only happens on the initial render

Reply:
i thought we would just add it as jonas says XD